main.py

```
@@ -1,228 +1,560 @@
+        if filter_arg == "unshared":
+            filter_arg = "notshared"
+        pagenum = 1
+        while True:
+            try: 
+                projects = self.session.mystuff_projects(filter_arg, page=pagenum, sort_by="")
+                if not projects:
+                    break
+                print(f"Found {len(projects)} projects on page {pagenum} with filter '{filter_arg}'")
+                self.projects += projects
+                print(f"Current amount of projects so far: {len(self.projects)}")
+            except Exception as e:
```

Ruff (0.15.10)

[warning] Do not catch blind exception: `Exception`

(BLE001)